packages/api-client/src/createApiClient.test.ts

```diff
@@ -342,24 +342,18 @@ describe("createAPIClient", () => {
       },
     });
 
-    expect(contentTypeSpy.mock.calls).toMatchInlineSnapshot(`
-      [
-        [
-          {
-            "accept": "application/json",
-            "accept-encoding": "gzip, deflate",
-            "accept-language": "*",
-            "connection": "keep-alive",
-            "content-length": "0",
-            "host": "localhost:3610",
-            "sec-fetch-mode": "cors",
-            "sw-access-key": "123",
-            "sw-context-token": "456",
-            "user-agent": "node",
-          },
-        ],
-      ]
-    `);
+    expect(contentTypeSpy).toHaveBeenCalledTimes(1);
+    const headers = contentTypeSpy.mock.calls[0]?.[0];
+    expect(headers).toBeDefined();
+    expect(headers).toMatchObject({
```

Comment:
we're not checking anymore what test declares - that `multipart/form-data` is removed